gst/datarepo/gstdatareposink.c

```diff
@@ -789,7 +767,7 @@ gst_data_repo_sink_change_state (GstElement * element,
 
     case GST_STATE_CHANGE_PAUSED_TO_PLAYING:
       GST_INFO_OBJECT (sink, "PAUSED_TO_PLAYING");
-
+      gst_data_repo_sink_is_static_tensors (sink);
```

Comment:
Q. Why do you move checking static tensors to the state change function?

Reply:
Thank you for the review :smile:
Depending on whether it is a static function or not, the behavior in the render function is branched,
If the pipeline is set_caps, the static status no longer changes,
So I thought checking it every time a buffer came in was wasteful.